packages/server/src/AccountsServer.js

```diff
@@ -302,6 +320,31 @@ export class AccountsServer {
     const res = await this.db.setProfile(userId, { ...user.profile, ...profile });
     return res;
   }
+
+  async sendVerificationEmail(userId: string, address: string): Promise<void> {
+    const user = await this.db.findUserById(userId);
+    if (!user) {
+      throw new AccountsError('User not found', { id: userId });
+    }
+    // If no address provided find the first unverified email
+    if (!address) {
+      const email = find(user.emails, (e: string) => !e.verified);
+      address = email && email.address; // eslint-disable-line no-param-reassign
+    }
+    // Make sure the address is valid
+    if (!address || !includes(user.emails.map((email: string) => email.address), address)) {
+      throw new AccountsError('No such email address for user');
+    }
+    const token = await this.db.addEmailVerificationToken(userId, address);
```

Comment:
Right now there is this function that generate the token https://github.com/js-accounts/accounts/blob/master/packages/server/src/tokens.js#L5 and in verifyEmail function there is this call to the bdd (who must take the token as parameter btw) to find the user with the token